File: src/Servers/PresenceConnectionManager/src/Aggregate/Misc/Status/UserStatusInfo.cs
namespace UniSpy.Server.PresenceConnectionManager.Aggregate.Misc
{
    /// <summary>
    /// This status info class is for new SDK version
    /// </summary>
    public sealed class UserStatusInfo
    {
        public string StatusState { get; set; }
        public string BuddyIP { get; set; }
        public string HostIP { get; set; }
        public string HostPrivateIP { get; set; }
        public int? QueryReportPort { get; set; }
        public int? HostPort { get; set; }
        public int? SessionFlags { get; set; }
        public string RichStatus { get; set; }
        public string GameType { get; set; }
        public string GameVariant { get; set; }
        public string GameMapName { get; set; }
        public string QuietModeFlags { get; set; }
        public UserStatusInfo()
        {
        }
    }
}

File: src/servers/presence_connection_manager/src/aggregates/user_status_info.py

from pydantic import BaseModel


class UserStatusInfo(BaseModel):
    status_state: str
    buddy_ip: str
    host_ip: str
    host_private_ip: str
    query_report_port: int
    host_port: int
    session_flags: str
    rich_status: str
    game_type: str
    game_variant: str
    game_map_name: str
    quiet_mode_flags: str

    def __init__(self) -> None:
        pass
